### DISCUSSION TITLE
Is it possible to buy a self-hosting slides.com editor licence?

### DISCUSSION THREAD

**Discussion by ftessier** [hakimel/reveal.js#3508] [Q&A]
I have been using slides.com and reveal.js for the last 3 years. It is fantastic! I would like to promote its use to create internal presentations within the large organization for which I work. However, the issue is that cloud services are prohibited for most presentation material, precluding using the slides.com editor.

I would like to know if it is possible to purchase a licence to self-host a slides.com editor internally inside the organization.

(I understand I can use reveal.js offline, or download the zip and then edit, and have been using these workarounds. However this is not practical for elaborate presentations where the slides.com "gui" proves essential.)

**Comment by hakimel** [comment]
Hey! We don't offer a self-hosted version of Slides today but are strongly considering it. We've drawn up initial plans and are trying to learn more from potential customers to ensure we're building the right thing.

- Would you want to self-host only the editor or are you looking for a more complete app, including user pages for listing all decks and a page for presenting?
- Are imports, exports, and being able to collaborate critical features to you in a self-hosted editor?
- Approximately how many people would have access to this in your organization?

Anything you can share about what the ideal version of a self-hosted Slides looks like to you would be very helpful.

**Reply by ftessier** [reply]
Good to hear! The minimal set of features at our end would be, in order of importance:

- Editor as on slides.com (especially css editor, equations and media library)
- ZIP export
- Collaboration

Of course, user account pages (deck listing etc.), as well as the speaker and live view would be great, in order to share and serve presentations internally in the organization (instead of feeding it through a meeting platform, where video can be choppy and lagging).

I would be happy to try any self-hosting implementation, step by step, and provide feedback. As far as I am concerned, even just the basic editor would be immediately useful!

The organization comprises ~4000 employees, but presumably we would start with a small set of keen users and add interested people to the roster as we go.

**Reply by jcralbino** [reply]
Hello @hakimel 

I wanted to know if there is any news in a OnPrem version of slide.com, and if there is something we can test .

I want to be able to host the presentations on our environment, and improve collaboration between teams by allowing them to edit and host their presentations 

It would be interesting to have the possibility of using a ci/cd pipeline that could validate changes on those presentations also and import presentations directly from git repository 

Do you have any recommended container/docker deployment?

Regards 
Jose

**Reply by jcralbino** [reply]
Hello @hakimel 

Do you have any updates on this topic?
